File: Deberes/igualdad_matrices.ts
const matriz1 = [
    [1, 2, 3],
    [4, 5, 6],
]
const matriz2 = [
    [1, 2, 3],
    [4, 5, 6],
    [7, 8 ,9]
]
let iguales = false
let m1 = matriz1.length
let m2 = matriz2.length
let n1 = matriz1[0].length
let n2 = matriz2[0].length
let j = 0
if (m1 == m2 && n1 == n2){
    do{
        for(let i = 0; i <= n1-1; i++){
            if(matriz1[j][i] == matriz2[j][i]){
                iguales = true
            } else{
                iguales = false
            }
         } 
         j++
    }while(j <= m1 -1)
    
    console.log(iguales)
}else{
    console.log(iguales)
}

 

File: Deberes/Deber 6/igualdad_matrices.py
matriz1 = [
    [1, 2, 4],
    [4, 5, 6],
    [7, 9, 6]
]
matriz2 = [
    [1, 2, 4],
    [4, 5, 6],
    [7, 8, 6]
]
iguales = True
m1 = len(matriz1)
m2 = len(matriz2)
n1 = len(matriz1[0])
n2 = len(matriz2[0])
i = 0
if m1 == m2 and n1 == n2:
    while i < m1 and iguales == True:
        for j in range(n1) :
            if matriz1[i][j] == matriz2[i][j]:
                iguales = True
            else:
                iguales = False   
                break                    
        i = i + 1
    if iguales :
        print("Las matrices son iguales")
    else : 
        print("Las matrices no son iguales")          
else:
    print("Las matrices no son iguales porque tienen tamaños diferentes")    
